x-pack/plugin/esql/qa/testFixtures/src/main/resources/mv_median_absolute_deviation.csv-spec

```diff
@@ -0,0 +1,82 @@
+example
+required_capability: fn_mv_median_absolute_deviation
+
+// tag::example[]
+ROW values = [0, 2, 5, 6]
+| EVAL median_absolute_deviation = MV_MEDIAN_ABSOLUTE_DEVIATION(values), median = MV_MEDIAN(values)
+// end::example[]
+;
+
+// tag::example-result[]
+values:integer | median_absolute_deviation:integer | median:integer
+[0, 2, 5, 6]   | 2                                 | 3
+// end::example-result[]
+;
+
+fromIndex
+required_capability: fn_mv_median_absolute_deviation
+
+FROM employees
+| WHERE emp_no <= 10002
+| SORT emp_no ASC
+| EVAL
+    int = MV_MEDIAN_ABSOLUTE_DEVIATION(salary_change.int),
+    long = MV_MEDIAN_ABSOLUTE_DEVIATION(salary_change.long),
+    double = MV_MEDIAN_ABSOLUTE_DEVIATION(salary_change)
+| KEEP emp_no, int, long, double
```

Comment:
I, as a reviewer, would like to see what values led to the said mv_mad calculus.

```suggestion
| KEEP emp_no, *int, *long, *double
```

Reply:
Added the input field to the `keep` as a reference